ansible/pbTestScripts/qemuPlaybookCheck.sh

```diff
@@ -71,9 +71,11 @@ defaultVars() {
 		"s390x" | "S390X" | "S390x" )
 			echo "s390x selected"; ARCHITECTURE=S390X;;
 		"aarch64" | "arm64" | "ARM64" )
-                        echo "aarch64 selected"; ARCHITECTURE=AARCH64;;
+      echo "aarch64 selected"; ARCHITECTURE=AARCH64;;
```

Comment:
Was this formatting change intentional? Looks like a mistake :-)

Reply:
Hmm, I tried fixing this but the indentation still came out a little off

Reply:
It looks like you've got 1 tab and then spaces in there- It should be 3 tabs